src/cluster/index.js

```diff
@@ -434,7 +398,7 @@ module.exports = class Cluster {
 
       topicConfigurations[topic] = { timestamp }
 
-      keys(partitionsPerLeader).map(nodeId => {
+      keys(partitionsPerLeader).forEach(nodeId => {
```

Comment:
This is my pet-peeve, so thank you 😄  I'm not gonna `git blame` this line, but just assume that Tulio wrote it and not me. 😉